pages/components/LinkedButton.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { ComponentProps, MouseEventHandler, ReactPropTypes } from "react";
+
+export type ButtonTypes = {
```

Comment:
calling this type ButtonTypes duplicates the context of "Types". We already know that it is a type because it is declared via the `type` keyword. I'd suggest postfixing types used for component props with "Props".

```suggestion
export type ButtonProps = {
```